clarify sentence (#559)
cds-hooks/docs: docs/specification/current.md

> Note that this means that CDS services will receive only the information they have requested and are authorized to receive. Prefetch data for other services registered to the same hook MUST NOT be provided. In other words, services SHALL only receive the data they requested in their prefetch and for which they are authorized recipients.

> The resulting response, which MUST be a single bundle that contains all information and has no "next page" links, is passed along to the CDS Service using the `prefetch` parameter (see [below](#example-prefetch-templates) for a complete example).

Reviewer: Not all prefetch response are Bundles (e.g. "Patient/{{context.patientId}}").  I'm open to other wordings, but here's an alternative:
```suggestion
The resulting response, which MUST be the entirety of the response (e.g. no "next page" links on a Bundle), is passed along to the CDS Service using the `prefetch` parameter (see [below](#example-prefetch-templates) for a complete example).
```
